coalib/misc/Exceptions.py

```diff
@@ -15,6 +17,12 @@ def get_exitcode(exception, log_printer=None):
         exitcode = 0
     elif isinstance(exception, SystemExit):
         exitcode = exception.code
+    elif (isinstance(exception, ContextualVersionConflict) or
+          isinstance(exception, VersionConflict)):
+        log_message = Constants.VERSION_CONFLICT_MESSAGE
+        log_message += exception.report()
+        log_printer.log_exception(log_message, exception)
+        exitcode = 4
```

Comment:
It's not really new: [Docs](http://coala.readthedocs.org/en/latest/Users/Exit_Codes.html?highlight=4) - Just felt that it's another version related problem. But I think this deserves its own exit code. How [about 13](http://www.ashout.com/fix-java-started-returned-exit-code13-eclipse/)? It's used by Java.